src/groups/GroupActions.js

```diff
@@ -10,7 +10,6 @@ export const QUERYSET_REQUEST_FAILURE = 'QUERYSET_REQUEST_FAILURE';
 export const QUERYSETS_RECEIVED = 'QUERYSETS_RECEIVED';
 export const QUERYSET_RECEIVED = 'QUERYSET_RECEIVED';
 
-export const REQUEST_FILTER_RANGES = 'REQUEST_FILTER_RANGES';
 export const RECEIVE_FILTER_RANGES = 'RECEIVE_FILTER_RANGES';
```

Comment:
we should probably nuke this line too to avoid confusion. that's probably my fault.